Using CFLAGS breaks compilation

If CFLAGS or LIBS is passed as a parameter to make the compilation will likely fail.

```
$ make CFLAGS="-g"
cc -g   -c -o conmon.o conmon.c
conmon.c:15:18: fatal error: glib.h: No such file or directory
 #include <glib.h>
                  ^
compilation terminated.
<builtin>: recipe for target 'conmon.o' failed
make: *** [conmon.o] Error 1
```

This happens because the new CFLAGS replaces the internal one where includes are being set.

--- a/conmon/Makefile
+++ b/conmon/Makefile
@@ -1,8 +1,8 @@
 src = $(wildcard *.c)
 obj = $(src:.c=.o)
 
-LIBS = $(shell pkg-config --libs glib-2.0)
-CFLAGS = -Wall -Wextra $(shell pkg-config --cflags glib-2.0)
+override LIBS += $(shell pkg-config --libs glib-2.0)
+override CFLAGS += -Wall -Wextra $(shell pkg-config --cflags glib-2.0)
 
 conmon: $(obj)
 	$(CC) -o $@ $^ $(LIBS)

